triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/PrettyPrint.scala

```diff
@@ -89,4 +92,90 @@ object PrettyPrint {
         }
         text("GenMap(") + intercalate(text(", "), list) + text(")")
     }
+
+  def prettySValue(v: SValue): Doc = v match {
```

Comment:
Trigger user state and messages are presented as `SValue` instances - hence the more user focused pretty printer